src/components/KioskLayout.tsx

```
@@ -0,0 +1,801 @@
+    <div className={styles.container}>
+      {/* Header */}
+      <Header title={{ en: "Buy Packages", es: "Comprar Paquetes" }} />
+
+      {/* Tab Navigation with Three Tabs */}
+      <div className={`${styles.tabsContainer} mt-16`}>
+        <div className={styles.tabs}>
+          <div 
+            className={`${styles.tab} ${activeTab === 'matpass' ? styles.active : styles.inactive}`} 
+            onClick={() => setActiveTab('matpass')}
+          >
+            MatPass
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)